moonworm/cli.py

```diff
@@ -100,8 +101,20 @@ def handle_brownie_generate(args: argparse.Namespace):
     with open(build_file_path, "r") as ifp:
         build = json.load(ifp)
 
+    relpath = os.path.relpath(project_directory, args.outdir)
+    splitted_relpath = [
```

Comment:
An alternative is to loop a function like [`os.path.split`](https://docs.python.org/3/library/os.path.html#os.path.split) than to explicitly split on `os.sep`. But I think this should be fine.